core/modules/replica/AdminApp.cc

```diff
@@ -0,0 +1,296 @@
+    vector<string> numNewRequests;
+    vector<string> numInProgressRequests;
+    vector<string> numFinishedRequests;
+
+    for (auto const& ptr: tracker.requests) {
+
+        workerName.push_back(ptr->worker());
+
+        if ((ptr->state()         == Request::State::FINISHED) &&
+            (ptr->extendedState() == Request::ExtendedState::SUCCESS)) {
+
+            startedSecondsAgo.push_back(    to_string((PerformanceUtils::now() - ptr->getServiceState().startTime) / 1000));
+            state.push_back(                                                     ptr->getServiceState().state2string());
+            numNewRequests.push_back(       to_string(                           ptr->getServiceState().numNewRequests));
+            numInProgressRequests.push_back(to_string(                           ptr->getServiceState().numInProgressRequests));
+            numFinishedRequests.push_back(  to_string(                           ptr->getServiceState().numFinishedRequests));
+
+        } else {
+            startedSecondsAgo.push_back(    "*");
+            state.push_back(                "*");
+            numNewRequests.push_back(       "*");
+            numInProgressRequests.push_back("*");
+            numFinishedRequests.push_back(  "*");
+        }
+    }
+    util::ColumnTablePrinter tableWorkers("WORKERS:", "  ", _verticalSeparator);
+
+    tableWorkers.addColumn("worker",                workerName, util::ColumnTablePrinter::Alignment::LEFT);
+    tableWorkers.addColumn("started (seconds ago)", startedSecondsAgo);
+    tableWorkers.addColumn("state",                 state,      util::ColumnTablePrinter::Alignment::LEFT);
+    tableWorkers.addColumn("queued",                numNewRequests);
+    tableWorkers.addColumn("in-progress",           numInProgressRequests);
+    tableWorkers.addColumn("finished",              numFinishedRequests);
+
+    cout << "\n";
+    tableWorkers.print(cout, false, false);
+
+    if (_dumpRequestInfo) {
+
+        vector<string>   workerName;
+        vector<string>   requestId;
+        vector<string>   requestType;
+        vector<string>   queue;
+        vector<uint32_t> priority;
+        vector<string>   database;
+        vector<uint32_t> chunk;
+        vector<string>   sourceWorkerName;
+
+        auto analyzeRemoteRequestInfo = [&](string const& worker,
+                                            string const& queueName,
+                                            proto::ReplicationServiceResponseInfo const& info) {
+            workerName      .push_back(worker);
+            requestId       .push_back(info.id());
+            requestType     .push_back(proto::ReplicationReplicaRequestType_Name(info.replica_type()));
+            queue           .push_back(queueName);
+            priority        .push_back(info.priority());
+            database        .push_back(info.database());
+            chunk           .push_back(info.chunk());
+            sourceWorkerName.push_back(info.worker());
```

Comment:
The White Space is free and it's abundant. Seriously speaking, in this type of code I prefer to align the code like columns in a table. This way it's easier to track what goes where.

Ideally, it would be nice if the STD collections had the overloaded operator `<<` or may be `+=` to allow stuff like this:
```
priority       << info.priority();
database    << info.database();
chunk          << info.chunk();
sourceWorkerName << info.worker();
```